rust/src/storage/azure/mod.rs

```diff
@@ -219,30 +218,21 @@ impl StorageBackend for AdlsGen2Backend {
         let length = data.len() as i64;
 
         // TODO: Consider using Blob API again since it's just 1 REST call instead of 3
```

Comment:
Don't hesitate to create a [follow-up issue ](https://github.com/delta-io/delta-rs/issues/new/choose) and add the link in the codebase to track the work to do.

Reply:
Done! #568